plugins/chat/config/locales/server.en.yml

```
@@ -1,6 +1,7 @@
 en:
   site_settings:
     chat_enabled: "Enable the chat plugin."
+    enable_public_channels: "Enable public channels."
```

```suggestion
    enable_public_channels: "Enable public channels based on categories."
```

Just a minor suggestion, but I am not sure if people will know that public channels == category channels; at the same time we may have already had this wording somewhere else.